src/btree/bt_page.c

```diff
@@ -115,7 +115,7 @@ __wt_page_alloc(
     /* Increment the cache statistics. */
     __wt_cache_page_inmem_incr(session, page, size);
     (void)__wt_atomic_add64(&cache->pages_inmem, 1);
-    page->cache_create_gen = __wt_atomic_load64(&cache->evict_pass_gen);
+    page->cache_create_gen = __wt_atomic_load64(&S2C(session)->evict->evict_pass_gen);
```

Comment:
```suggestion
    (void)__wt_atomic_add64(&S2C(conn)->cache->pages_inmem, 1);
```
and we can drop the local `cache` variable

Reply:
Updated in [add50ca](https://github.com/wiredtiger/wiredtiger/pull/11037/commits/add50ca3177b3a8d182c20b2e499f52f2ee3b3d3)